tuning/automation/rocblas-benchInputCreator.py

```diff
@@ -215,38 +231,140 @@ def createYaml(args, outputfile, problem, sizeMappings, verify):
 
         sizeParams = getSizeParams(size, transA, transB)
 
+        latency = 2*sizeParams['M']*sizeParams['N']*sizeParams['K']/perf[1]/1000 # us
+        niters = math.ceil( 1.15 * 3e6 / latency)
+        
+        otherParams = {"alpha": 1, "beta": 1, "iters": niters, "cold_iters": niters}
+        otherParams_rotating = {"alpha": 1, "beta": 1, "iters": niters, "cold_iters": niters, "flush_memory_size": 536870812}
```

Comment:
NDB (no action is needed) const numbers might be better defined on top and not repeated

Reply:
done.